src/main/java/org/broadinstitute/consent/http/service/validate/UseRestrictionValidator.java

```diff
@@ -53,8 +53,8 @@ public void validateUseRestriction(String useRestriction) throws IllegalArgument
             if (!entity.isValid()) {
                 throw new IllegalArgumentException(entity.getErrors().toString());
             }
-        }else{
-            throw new IllegalArgumentException("There was an error posting the use restriction" + (res.readEntity(ValidateResponse.class)).getErrors().toString());
+        } else {
+            throw new IllegalArgumentException("There was an error posting the use restriction" + (res.readEntity(String.class)));
```

Comment:
This is not technically related to this PR, but this bug added all sorts of error messages to the logs which made debugging much harder than it needed to be.